config/crd/v1/bases/velero.io_backuprepositories.yaml

```diff
@@ -88,8 +88,8 @@ spec:
             description: BackupRepositoryStatus is the current status of a BackupRepository.
             properties:
               lastMaintenanceTime:
-                description: LastMaintenanceTime is the last time maintenance was
-                  run.
+                description: LastMaintenanceTime is the last time repo maintenance
```

Comment:
`LastMaintenanceTime` should be used to record the completion time of a successful maintenance.
Previously, it is used to record the start time of a successful maintenance, which is not reasonable. E.g., I run a maintenance  which takes more than 1 hour; since the default frequency for quick maintenance is 1 hour, the next maintenance will be launched immediately after the first one completes, but actually it is a wasting one as data has just been maintained.